src/Form/GCSettingsForm.php

```diff
@@ -144,6 +151,20 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
       '#placeholder' => '/path/to/page',
     ];
 
+    $form['app_settings']['live_stream_empty_text'] = [
```

Comment:
Please remove `$form['app_settings']['live_stream_empty_text']` and `$form['app_settings']['virtual_meeting_empty_text']`